isolation/_05_ReadCommitted_RepeatableRead/no_index_column/aggregate/insert_select_01_1.ctl

```diff
@@ -36,13 +36,15 @@ MC: wait until C1 ready;
 
 /* test case */
 C1: update t set id=col-1 where mod(id,2)=0;
+MC: wait until C2 ready;
```

Comment:
wait until C2 ready; 
to be
wait until C1 ready;

Reply:
It is intended.
The test case is expect to show C3's result first , then C2 , then C1.

Reply:
I think two ways are right.
What you said may be easier to understand.
I will modify it.